libsolidity/analysis/DocStringTagParser.h

```diff
@@ -37,6 +37,8 @@ class DocStringTagParser: private ASTConstVisitor
 public:
 	explicit DocStringTagParser(langutil::ErrorReporter& _errorReporter): m_errorReporter(_errorReporter) {}
 	bool parseDocStrings(SourceUnit const& _sourceUnit);
+	/// Validate the parsed doc strings, requires the resolver to have run
```

Comment:
You mean this resolver, right?
```suggestion
	/// Validate the parsed doc strings, requires the NameAndTypeResolver to have run
```

Reply:
Well, technically it is the ReferencesResolver which is in turned called by the NameAndTypeResolver

Reply:
Ah, my bad. All the more reason to include its name though :)

Reply:
Also, it probably also requires `parseDocStrings()` to run before it, right? If so I'd mention it in the docstring too.